frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -100,6 +124,63 @@ const TranslationPage = () => {
     );
   };
 
+  const onUserInput = async (newContent: string, lineIndex: number) => {
+    const oldContent =
+      translatedStoryLines[arrayIndex(lineIndex)].translatedContent;
+    if (oldContent !== undefined) {
```

Comment:
I believe you can skip this check if 129 has a [non null assertion operator](https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci)
something like
```
const oldContent =
      translatedStoryLines[arrayIndex(lineIndex)].translatedContent!;
```